### DISCUSSION TITLE
Running v6 locally

### DISCUSSION THREAD

**Discussion by lparsons** [super-linter/super-linter#5451] [Q&A] [answered]
Before pushing updates to our GitHub repo, we like to run super-linter locally. I've been doing that using a command like:
```
docker run -e RUN_LOCAL=true "github/super-linter:slim-v5"
```
However, there does not appear to be any docker images for version 6 of super-linter. I get error messages when I try `docker run` and I can't find anything at https://hub.docker.com/r/github/super-linter/tags. Is there a new way I should specify the version to run locally? Thanks.

**Comment by lparsons** [comment, marked as answer]
Answering my own question. I failed to notice the change in the reference to the image. The following seems to work fine: `ghcr.io/super-linter/super-linter:slim-v6.3.0`. Note that on must use `ghcr.io` instead of `github` when referencing the image.

**Reply by ferrarimarco** [reply]
Thanks for following up!
